tribuo-classification/README.md

```diff
@@ -177,8 +177,14 @@ Start Client 2 and pass training data to train the model
 Start Client 3 and evaluate the model
 
       java -jar client/build/libs/client-all.jar --role="EVALUATE"  --constraint="S:3D71AAC9ED596F3C86241A977DAFA84B255D16D633DFF7AB88A053AE2183AD14 PROD:1 SEC:INSECURE" --url="http://localhost:8080"
-   
 
+## How to generate the reflection and serialization configuration files for the enclave
+
+Conclave uses the GraalVM Native Image JVM, in which all bytecode is compiled to native code ahead of time. However, 
+this cannot always predict all usages of the Java Reflection or the Serialization.
+Undetected usages of these dynamic features must be provided to the enclave in configuration files.
+You can find these configuration files at tribuo-classification/enclave/src/main/resources/META-INF/native-image location.
```

Comment:
Let's format the location path with quotes `

Reply:
done